### DISCUSSION TITLE
Allow loading XDTS files as Sub-XSheets

### DISCUSSION THREAD

**Discussion by WolfInABowl** [opentoonz/opentoonz#6770] [Ideas]
XDTS files are loaded via the `Load Scene` dialog in the same manner as native OT scenes.
However, unlike native OT scenes, XDTS files cannot be loaded via the `Load As Sub-XSheet` command.
This limitation appears to stem from the fact that no actual scene files are created when loading an XDTS file, thus no scene is present to be loaded as a Sub-XSheet.

If the user wanted to import an XDTS file into an existing scene, the user must perform these steps:
1. Save the currently open scene (Scene A), if needed
2. Open the desired XDTS file and its associated image sequences
3. Save the imported XDTS data as a new scene (Scene B)
4. Reopen Scene A and load Scene B into it as a Sub-XSheet
Alternatively, another instance of OT can be used to perform steps 2 to 3 to avoid switching scenes.

I believe that a solution to this would be to allow the automatic creation of a native OT scene when importing an XDTS file.
This would allow the user to load the XDTS as a Sub-XSheet, as a native OT scene file with the XDTS data would be created in the background. This would turn the aforementioned 4-step process into a single action.

It may make sense to make this a toggle-able option as part of the XDTS import dialog, with the option forced on if the XDTS file is loaded as a Sub-XSheet.
Alongside this toggle would be two additional text-entry fields, the first allowing for specifying the target location for the scene file, and the second allowing the user to override the scene file's name (which would otherwise default to the XDTS file's name).

**Comment by RodneyBaker** [comment]
In addition to the criteria of this request, the locations where .tnz files can be loaded should likely also see and allow .xdts files to be loaded.
